interface/main/holidays/Holidays_Controller.php

```diff
@@ -0,0 +1,109 @@
+<?php
+require_once("Holidays_Storage.php");
+/**
+ * Created by PhpStorm.
+ * User: sharonco
+ * Date: 6/15/16
+ * This class contains the implementation of all the logic included in the holidays calendar story
+ */
+class Holidays_Controller{
+
+    const UPLOAD_DIR = "documents";
```

Comment:
Lets not dump these directly in the documents directory. Much better to create a special directory in the documents directory such as documents/holiday_storage/ .